File: Voronezh/Bars.Gkh.Regions.Voronezh/libs/B4/model/RosRegExtractDesc.js
Ext.define('B4.model.RosRegExtractDesc', {
    extend: 'B4.base.Model',
    idProperty: 'Id',

    proxy: {
        type: 'b4proxy',
        controllerName: 'RosRegExtractDesc'
    },
    fields: [
        { name: 'Id' },
        { name: 'Desc_ID_Object' },
        { name: 'Desc_CadastralNumber' },
        { name: 'Desc_ObjectType' },
        { name: 'Desc_ObjectTypeText' },
        { name: 'Desc_ObjectTypeName' },
        { name: 'Desc_AssignationCode' },
        { name: 'Desc_AssignationCodeText' },
        { name: 'Desc_Area' },
         { name: 'PersAcc' },        
        { name: 'Desc_AreaText' },
        { name: 'Desc_AreaUnit' },
        { name: 'Desc_Floor' },
        { name: 'Desc_ID_Address' },
        { name: 'Desc_AddressContent' },
        { name: 'Desc_RegionCode' },
        { name: 'Desc_RegionName' },
        { name: 'Desc_OKATO' },
        { name: 'Desc_KLADR' },
        { name: 'Desc_CityName' },
        { name: 'Desc_Urban_District' },
        { name: 'Desc_Locality' },
        { name: 'Desc_StreetName' },
        { name: 'Desc_Level1Name' },
        { name: 'Desc_Level2Name' },
         { name: 'Desc_ApartmentName' },
        { name: 'Reg_RegDate' },
        { name: 'Reg_RegNumber' },
         { name: 'Room_id' },
         { name: 'YesNoNotSet' }
        
    ]
});

File: Voronezh/Bars.Gkh.Regions.Voronezh/Entities/RosRegExtract/RosRegExtractDesc.cs
using Bars.B4.DataAccess;
using Bars.Gkh.Entities;
using Bars.Gkh.Enums;
using System;

namespace Bars.Gkh.Regions.Voronezh.Entities
{
    public class RosRegExtractDesc : PersistentObject
    {
        //Description
        public virtual string Desc_ID_Object { get; set; }
        public virtual string Desc_CadastralNumber { get; set; }
        public virtual string Desc_ObjectType { get; set; }
        public virtual string Desc_ObjectTypeText { get; set; }
        public virtual string Desc_ObjectTypeName { get; set; }
        public virtual string Desc_AssignationCode { get; set; }
        public virtual string Desc_AssignationCodeText { get; set; }
        public virtual string Desc_Area { get; set; }
        public virtual string Desc_AreaText { get; set; }
        public virtual string Desc_AreaUnit { get; set; }
        public virtual string Desc_Floor { get; set; }
        public virtual string Desc_ID_Address { get; set; }
        public virtual string Desc_AddressContent { get; set; }
        public virtual string Desc_RegionCode { get; set; }
        public virtual string Desc_RegionName { get; set; }
        public virtual string Desc_OKATO { get; set; }
        public virtual string Desc_KLADR { get; set; }
        public virtual string Desc_CityName { get; set; }
        public virtual string Desc_Urban_District { get; set; }
        public virtual string Desc_Locality { get; set; }
        public virtual string Desc_StreetName { get; set; }
        public virtual string Desc_Level1Name { get; set; }
        public virtual string Desc_Level2Name { get; set; }
        public virtual string Desc_ApartmentName { get; set; }

        /// <summary>
        /// Помещение
        /// </summary>
        public virtual Int64 Room_id { get; set; }

        /// <summary>
        /// Сопоставлен
        /// </summary>
        public virtual YesNoNotSet YesNoNotSet { get; set; }


        //Header fields
        public virtual string ExtractDate { get; set; }
        public virtual string ExtractNumber { get; set; }
        public virtual string HeadContent { get; set; }
        public virtual string Registrator { get; set; }
        public virtual string Appointment { get; set; }
        public virtual string NoShareHolding { get; set; }
        public virtual string RightAgainst { get; set; }
        public virtual string RightAssert { get; set; }
        public virtual string RightClaim { get; set; }
        public virtual string RightSteal { get; set; }
        //public virtual string XML { get; set; }
    }
}
